learners/setup.md

```diff
@@ -2,53 +2,33 @@
 title: Setup
 ---
 
-FIXME: Setup instructions live in this document. Please specify the tools and
-the data sets the Learner needs to have installed.
-
-## Data Sets
-
-<!--
-FIXME: place any data you want learners to use in `episodes/data` and then use
-       a relative link ( [data zip file](data/lesson-data.zip) ) to provide a
-       link to it, replacing the example.com link.
--->
-Download the [data zip file](https://example.com/FIXME) and unzip it to your Desktop
-
-## Software Setup
-
-::::::::::::::::::::::::::::::::::::::: discussion
-
-### Details
-
-Setup for different systems can be presented in dropdown menus via a `spoiler`
-tag. They will join to this discussion block, so you can give a general overview
-of the software used in this lesson here and fill out the individual operating
-systems (and potentially add more, e.g. online setup) in the solutions blocks.
-
-:::::::::::::::::::::::::::::::::::::::::::::::::::
-
-:::::::::::::::: spoiler
-
-### Windows
-
-Use PuTTY
-
-::::::::::::::::::::::::
-
-:::::::::::::::: spoiler
-
-### MacOS
-
-Use Terminal.app
-
-::::::::::::::::::::::::
-
-
-:::::::::::::::: spoiler
-
-### Linux
-
-Use Terminal
-
-::::::::::::::::::::::::
-
+## Install R and RStudio
+  * Use the following to find installation instructions under “Preparations”: [https://datacarpentry.github.io/R-ecology-lesson/index.html#preparations](https://datacarpentry.github.io/R-ecology-lesson/index.html#preparations)  
+      
+* Follow the instructions to install the "tidyverse" and "ratdat" packages:  
+  * In the RStudio Console enter: 
+  install.packages("tidyverse")
+  install.packages("ratdat")
```

Comment:
```suggestion
* Install the "tidyverse" and "ratdat" packages. In the RStudio Console enter: 
  install.packages("tidyverse")
  install.packages("ratdat")
```

Reply:
I'd like to format the install code as a code block but I can't do it in a comment here. So I'll do it once I accept my changes.